firmware/src/onboarding_flow/onboarding_flow.cpp

```diff
@@ -1,49 +1,8 @@
 #include "onboarding_flow.h"
 #include "apps/icons.h"
-#include "qrcode.h"
 
-OnboardingFlow::OnboardingFlow()
+OnboardingFlow::OnboardingFlow(TFT_eSprite *spr_, TFT_eSprite qrcode_spr_) : spr_(spr_), qrcode_spr_(qrcode_spr_)
 {
-
-    root_level_motor_config = PB_SmartKnobConfig{
```

Comment:
Why did you remove this?

Reply:
nevermind, found my answer

Reply:
TFT_eSprite doesnt have a default constructor and either needs a reference to tft_ (TFT_eSpi) or to be initialized from the start.